met/src/tools/other/pb2nc/pb2nc.cc

```diff
@@ -3088,7 +3090,7 @@ void insert_pbl(float *obs_arr, const float pbl_value, const int pbl_code,
    hdr_info << unix_to_yyyymmdd_hhmmss(hdr_vld_ut)
             << " " << hdr_typ << " " << hdr_sid;
    if (is_eq(pbl_value, bad_data_float)) {
```

Comment:
Recommend calling "is_bad_data(float)" function instead which also checks for NaN.
```suggestion
   if (is_bad_data(pbl_value)) {
```